src/Documents/Export/PageExporter.php

```diff
@@ -32,7 +32,11 @@ public function toYaml(Page $page): string
         return $this->serializer->serialize(
             [self::PAGE => $yamlExportPage],
             'yaml',
-            ['yaml_inline' => 4, 'yaml_indent' => 4, 'yaml_flags' => self::YAML_DUMP_FLAGS],
+            [
+                'yaml_inline' => 4, // bis zu welcher Ebene des Array/Objekts soll YAML erzeugt werden
+                'yaml_indent' => 0, // Wieviel Einrückung soll das YAML initial kriegen?
```

Comment:
This is the default setting anyway, so we can omit it.

Otherwise: in English please ;)